packages/example-app-base/src/components/Page/Page.module.scss

```diff
@@ -78,8 +79,10 @@ $PageSubHeadMargin: 20px;
     @include ms-fontSize-16;
   }
 
-  p {
-    margin-bottom: 12px;
+  p,
+  pre {
+    margin-top: 0;
+    margin-bottom: 1.2em;
```

Comment:
@jordandrako Let me know what you think of this new spacing--at least to me it seemed to give more readable results overall, especially on dense pages like getting started.

Reply:
I agree, but I suspect we'll have to get this approved.